iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/HvdcLineAdderAdapter.java

```diff
@@ -0,0 +1,180 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.mergingview;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.HvdcLine;
+import com.powsybl.iidm.network.HvdcLineAdder;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.util.Identifiables;
+
+import java.util.Objects;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class HvdcLineAdderAdapter implements HvdcLineAdder {
+
+    private String id;
+
+    private boolean ensureIdUnicity = false;
+
+    private String name;
+
+    private double r;
+
+    private HvdcLine.ConvertersMode convertersMode;
+
+    private double nominalV;
+
+    private double maxP;
+
+    private double activePowerSetpoint;
+
+    private String converterStationId1;
+
+    private String converterStationId2;
+
+    private MergingViewIndex index;
+
+    HvdcLineAdderAdapter(final MergingViewIndex index) {
+        this.index = Objects.requireNonNull(index, "merging view index is null");
+    }
+
+    @Override
+    public HvdcLine add() {
+        final Network n1 = checkAndGetNetwork1();
+        final Network n2 = checkAndGetNetwork2();
+        if (n1 != n2) {
+            throw new PowsyblException("HvdcLine creation between two networks is not allowed");
+        }
+        checkAndSetUniqueId();
+        return index.getHvdcLine(addLine(n1));
+    }
+
+    private Network checkAndGetNetwork1() {
+        if (Objects.isNull(converterStationId1)) {
+            throw new PowsyblException("Side 1 converter station is not set");
+        }
+        final Network network = index.getNetworkStream()
+                                     .filter(n -> n.getHvdcConverterStation(converterStationId1) != null)
```

Comment:
The merging view should implement `getHvdcConverterStation(String)`: why don't you use it?

Reply:
We need to search which network contains "converterStationId1" & "converterStationId1"
In order to know, if we are creating HvdcLine between 2 networks or not.
For now, no creation are allowed between 2 networks
So, i need to get reference on networks contains into mergingview